docs/scos/dev/guidelines/keeping-a-project-upgradable/keeping-a-project-upgradable.md

```diff
@@ -6,8 +6,6 @@ template: concept-topic-template
 related:
   - title: Upgrader tool overview
     link: docs/scos/dev/guidelines/keeping-a-project-upgradable/upgrader-tool-overview.html
```

Comment:
The article is moved and renamed. Please update `url` and `title`

Reply:
The same is here. I used the master branch and didn't change it, so why should I update something here?